functions/pipeline/shared/db_access/db_access_v2.py

```diff
@@ -130,19 +130,16 @@ def add_new_images(self,list_of_image_infos, user_id):
             finally: conn.close()
         return url_to_image_id_map
 
-    def get_tag_complete_images(self, number_of_images, user_id):
-        if number_of_images <= 0:
-            raise ArgumentException("Parameter must be greater than zero")
-
+    def get_tag_complete_images(self, user_id):
         tag_complete_images = {}
         try:
             conn = self._db_provider.get_connection()
             try:
                 cursor = conn.cursor()
                 query = ("SELECT b.ImageId, b.ImageLocation, a.TagStateId FROM Image_Tagging_State a "
-                        "JOIN Image_Info b ON a.ImageId = b.ImageId WHERE a.TagStateId = {1} order by "
-                        "a.createddtim DESC limit {0}")
-                cursor.execute(query.format(number_of_images, ImageTagState.COMPLETED_TAG))
```

Comment:
Not limiting could cause long runtimes over the limit (300 secs?) or cause the final return payloads to grow over what is allowed through our API gateway.

Reply:
Right, we decided to use all images for training (for now) so that's why I removed the limit, @andrebriggs should I bring a limit back?

Reply:
@samiyaakhtar sure for now let's set a limit for now. Maybe choose 200 or something